test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_mnist.py

```diff
@@ -219,20 +256,19 @@ def test_smdataparallel_smmodelparallel_mnist(sagemaker_session, instance_type,
     estimator.fit()
 
 
-def _assert_checkpoint_exists_v2(s3_model_dir):
+def _assert_checkpoint_exists_v2(region, model_dir, checkpoint_number):
```

Comment:
nit: can you clearly describe what you mean by v2 style checkpoints? Assuming this doesn't mean tensorflow2, since all of these tests run on TF2